src/connection.ts

```diff
@@ -891,6 +891,12 @@ class Connection extends EventEmitter {
    * @private
    */
   fedAuthInfoToken: undefined | FedAuthInfoToken;
+  /**
+   * @private
+   * Set when FeatureExtAck for server UTF-8 support is received.
+   * Applicable only when tdsVersion >= 7.4.
+   */
+  utf8Support?: boolean;
```

Comment:
Would it make sense to default this to `false`? 🤔

Reply:
i was making a three-way distinction of n/a, true, false. but i'm tempted to not save it at all; i don't know that we'll ever actually need to refer to it once we turn it on.

Reply:
further argument in favor of dropping the commit that saves that in the connection: mssql-jdbc doesn't save it, either.

https://github.com/microsoft/mssql-jdbc/blob/0c007b5f678f26f00b84e28895ee449dafd753a9/src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java#L5041-L5050